### DISCUSSION TITLE
Create Cluster Group as the VC name not Datacentre name

### DISCUSSION THREAD

**Discussion by chrisclarke6710** [bb-Ricardo/netbox-sync#150] [Q&A]
I'm testing the Netbox_sync script and does a great job on overall.  
However Currently the Cluster Group is linked to the Datacentre object name within the VC.
I understand the reasoning for this however, it would be good to have the option to create Cluster Groups based on the VC name rather than Datacentre name.  
Reason:  
  When supporting multiple VC servers in the same DC (used for different purposes) all configured to use the same Datacentre name, it becomes very confusing to which VC a VM and Cluster belongs too.

Ideally the option to have the Cluster Group created as the VC name, and then the datacentre name as custom attribute of the Cluster.  

Many thanks

**Comment by bb-Ricardo** [comment]
Good idea. Can you please open an issue for that?
